ml-agents/mlagents/trainers/learn.py

```diff
@@ -49,6 +50,23 @@ def parse_command_line(argv: Optional[List[str]] = None) -> RunOptions:
     return RunOptions.from_argparse(args)
 
 
+def _Get_checkpoint_name(
+    behavior_name: str, checkpoint_list: Optional[Dict[str, str]]
+) -> str:
+    if checkpoint_list and checkpoint_list.get(behavior_name):
+        return checkpoint_list[behavior_name]
+    else:
+        return "checkpoint.pt"
```

Comment:
This is a constant, I think it is hard coded somewhere else in the `torch_model_saver.py` script. Please use a constant so we can change it at only one place.